scripts/Muon/table_utils.py

```diff
@@ -53,7 +53,7 @@ def setTableHeaders(table):
         if os.name != "nt":
             return
         version=QtCore.QSysInfo.WindowsVersion
-        windows10=160 # version code?
+        windows10=QtCore.QSysInfo.WindowsVersion
         if(version==windows10):
```

Comment:
I feel like I'm missing something but is this not comparing something with itself now?

Reply:
yes you are right. I will keep looking for a different answer